src/main/java/Duke.java

```diff
@@ -1,10 +1,249 @@
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Scanner;
+
 public class Duke {
+    private final String name;
+    private final List<Task> tasks;
+
+    public Duke(String name) {
+        this.name = name;
+        this.tasks = new ArrayList<>();
+    }
+
+    public void greet() {
+        System.out.println("=========================");
+        System.out.println("Hello! I'm " + name);
+        System.out.println("What can I do for you?");
+    }
+
+//    public void addTask(String description) {
```

Comment:
Avoid redundant commented codes, especially when there is an existing function with the same name